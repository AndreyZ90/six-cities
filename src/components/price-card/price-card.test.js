import React from 'react';
import TestRenderer from 'react-test-renderer';

import PriceCard from '@/components/price-card/price-card';

const price = 300;

describe(`PriceCard component snapshot`, () => {
  test(`Should correctly render PriceCard component`, () => {
    const tree = TestRenderer.create(<PriceCard price={price} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
