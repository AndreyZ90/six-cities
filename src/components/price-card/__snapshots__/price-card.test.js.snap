// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PriceCard component snapshot Should correctly render PriceCard component 1`] = `
<div
  className="place-card__price"
>
  <b
    className="place-card__price-value"
  >
    €
    300
  </b>
  <span
    className="place-card__price-text"
  >
     / night
  </span>
</div>
`;
